### PR TITLE
GG-20804 PME test optinization

### PR DESCRIPTION
**Pull request by ingvard**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/cache/store/jdbc/CacheJdbcPojoWriteBehindStoreWithCoalescingTest.java [hunk 1]
```diff
@@ -552,7 +557,7 @@ public void readWithCoalescing() throws Exception {
             @Override public void run() {
                 try {
                     while (t2Count.get() > 0) {
-                        for (int i = 200000; i < 400000; i++) {
+                        for (int i = stepSize; i < 2* stepSize; i++) {
```
Comment (dspavlov) [inline modules/core/src/test/java/org/apache/ignite/cache/store/jdbc/CacheJdbcPojoWriteBehindStoreWithCoalescingTest.java]: 2 * <-spac is missing between 2 and *

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/CacheLateAffinityAssignmentTest.java [hunk 2]
```diff
@@ -2134,7 +2134,7 @@ public void testNoForceKeysRequests() throws Exception {
 
         final AtomicInteger nodeIdx = new AtomicInteger();
 
-        final long stopTime = System.currentTimeMillis() + 60_000;
+        final long stopTime = System.currentTimeMillis() + GridTestUtils.SF.applyUB(400_000, 60_000);
```
Comment (dspavlov) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/CacheLateAffinityAssignmentTest.java]: It seems test stop time is raised here to 400 seconds. Should we reduce it?

Comment (ingvard) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/CacheLateAffinityAssignmentTest.java]: No, this means that max time 60 seconds and min 400_000 * 0.1 miles = 40 seconds.

File: modules/core/src/test/java/org/apache/ignite/cache/store/jdbc/CacheJdbcPojoWriteBehindStoreWithCoalescingTest.java (hunk 1 above)
Comment (ingvard) [reply modules/core/src/test/java/org/apache/ignite/cache/store/jdbc/CacheJdbcPojoWriteBehindStoreWithCoalescingTest.java]: fixed

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/CacheLateAffinityAssignmentTest.java (hunk 2 above)
Comment (ingvard) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/CacheLateAffinityAssignmentTest.java]: fixed

**Pull request closed without merging**